apps/web/pages/auth/login.tsx

```diff
@@ -57,7 +57,7 @@ export default function Login({
         .string()
         .min(1, `${t("error_required_field")}`)
         .email(`${t("enter_valid_email")}`),
-      password: z.string().min(1, `${t("error_required_field")}`),
+      password: !!totpEmail ? z.literal("") : z.string().min(1, `${t("error_required_field")}`),
```

Comment:
@alannnc thanks for the suggestion, here it is, if we are in a TOTP code context, we allow empty password, otherwise apply the minimum limit